assets/javascripts/modules/autoComplete.js

```diff
@@ -339,6 +346,8 @@ var setup = function ($elem, suggestionsData, $targetInputElem, suggestionFormat
     $clearInputButton = $('.js-suggestions-clear');
     $suggestionsContainer = $('.js-suggestions').first();
     $suggestionsStatusMessage = $('.js-suggestions-status-message').first();
+    $hiddenInput = $('<input type="hidden" id="hiddeninput" />')
```

Comment:
I'd suggest storing the selected title and value in one or two data-attributes on the autocomplete input - rather than creating a new input - keeps it a little more central and easier to query later on if needed.